app/views/ViewUtils.scala

```diff
@@ -45,6 +45,11 @@ object ViewUtils {
     LocalDateTime.from(instant.atZone(ZoneId.systemDefault())).format(formatter)
   }
 
+  def dateTime(instant: Instant): String = {
```

Comment:
Can you write a test for the method? It also might be good if the name of the method is named in a more specific way if possible. Current name is very general, while the formatting seems specific.